<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Banking transactions</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="dateLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="dateDisplay">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>textlabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="amountLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Amount</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="amountDoubleSpinBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="depositButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>150</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Deposit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="withdrawalButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>150</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Withdrawal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="toFileButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>150</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>To file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sortButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>160</y>
      <width>150</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort</string>
    </property>
   </widget>
   <widget class="QComboBox" name="sortComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>150</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Selection</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Insertion</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
